### PR TITLE
Updating Animated text

### PR DESCRIPTION
**Pull request by Duncid**
## Description
Largely refactor Animated text so it looks better and fit our needs!

### CODE REVIEW

File: sparkle/src/components/AnimatedText.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { cva, VariantProps } from "class-variance-authority";
+import { CSSProperties, ReactNode } from "react";
+import React from "react";
+
+import { cn } from "@sparkle/lib/utils";
+
+const animVariants = cva(
+  "s-relative s-mx-auto s-max-w-md s-text-black/0 s-animate-shiny-text s-bg-clip-text s-bg-no-repeat [background-position:0_0] [background-size:var(--shiny-width)_100%] s-bg-gradient-to-r",
```
Comment (philipperolet) [inline sparkle/src/components/AnimatedText.tsx:8]:
```suggestion
  "s-relative s-mx-auto s-max-w-md s-text-black/0 s-animate-shiny-text s-bg-clip-text s-bg-no-repeat [background-position:0_0] [background-size:50%_100%] s-bg-gradient-to-r",
```

File: sparkle/src/components/AnimatedText.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+export function AnimatedText({
+  children,
+  variant,
+  className,
+}: AnimatedShinyTextProps) {
+  return (
+    <span className={cn("s-relative s-inline-block", className)}>
+      <span className={textVariants({ variant })}>{children}</span>
+      <span
+        style={
```
Comment (philipperolet) [inline sparkle/src/components/AnimatedText.tsx:51]: the style here is actually useless & not good practice; Iorgot to remove it in my former PR. See the change right above, cleaner this way

**Review by philipperolet — COMMENTED** [review]
:+1: 
Left a little nit, then good to merge

**Comment by Duncid** [comment]
Thanks @philipperolet !

**Review by philipperolet — APPROVED** [review]

**Pull request merged**